Islandora/islandora_solution_pack_newspaper
# ISLANDORA-2116 adjusted newspaper display to only load first year iss…

…ues, also added ajax handler to populate for any year clicked

**JIRA Ticket**: (link)
https://jira.duraspace.org/browse/ISLANDORA-2116

# What does this Pull Request do?

This will change the newspaper view so that only the first year's issues are loaded into the web browser -- and all clicks to other years are handled by an AJAX call.

# What's new?
This also adds a thumbnail to the view area and automatically expands the loaded fieldsets so that they are not collapsed by default - which may not be desired by some, but our usability study has asked why they were collapsed (I could remove that modification from this pull request).

This would change what gets indexed by search engines for the initial load of the page.

# How should this be tested?

A thorough test would be to:
1. (before merging this code), load a newspaper into the browser that has more than one year of issues and preferably thousands of issues.  Inspect the load time for this web page.
2. Install the updated code here and clear the drupal cache.  Repeat step 1 to see how the load time should be much quicker with far fewer requests (only the first year should be loaded into the browser initially).
3. ensure that each year can be clicked and loaded into the right side of the screen where the issues are displayed.

# Additional Notes:
As I do not see any specific documentation relative to the newspaper viewer, there probably isn't any need to change the wiki documentation.
